falcon/request.py

```diff
@@ -514,6 +516,35 @@ def cookies(self):
 
         return self._cookies.copy()
 
+    @property
+    def access_route(self):
+        """If a forwarded header exists this is a list of all ip addresses
```

Comment:
Style: First line of a comment should be short enough to avoid wrapping. Additional documentation can be placed in subsequent paragraphs, separated by a blank line.